<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Rmse</string>
     </attribute>
     <widget class="QTextEdit" name="rmse_view">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RMSE</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="r2_view">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RSCORE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>80</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>pred_time</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="pred_time">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>120</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>y_pred</string>
     </attribute>
     <widget class="QTableWidget" name="ypred_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>541</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>쪽</string>
     </attribute>
     <widget class="QLineEdit" name="csv_name">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>331</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="csv_path_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>csv 저장 경로</string>
      </property>
     </widget>
     <widget class="QPushButton" name="csv_name_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>420</y>
        <width>331</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>GET TEXT</string>
      </property>
     </widget>
     <widget class="QListWidget" name="csv_path_label">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>331</width>
        <height>131</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="csv_store">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>340</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>csv저장</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>600</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>종료</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
